templates/zerver/help/add-a-bot-or-integration.md

```diff
@@ -4,7 +4,7 @@ Bots and integrations are features that are accessible to all members of an orga
 
 ## Add a bot
 
-{!go-to-the.md!} [Your bots](/#settings/your-bots)
+1. Go to the [Your bots](/#settings/your-bots)
```

Comment:
These were all found and replaced with the command `
find templates/zerver/help -type f -exec sed -i 's/{!go-to-the\.md!}/1\. Go to the/g' {} \;`